### PR TITLE
Update workflow schema/values and bump to 0.2.3

### PR DESCRIPTION
**Pull request by masayag**

### CODE REVIEW

File: charts/workflows/charts/mta/values.yaml [hunk 1]
```diff
@@ -1,7 +1,3 @@
 mta:
   # specify the port because of rest client bug in sonata (TODO check when is this solved)
-  endpoint: https://mta-openshift-mta.apps.parodos-dev.projects.ecosystem.sysdeseng.com:443/hub/ 
-
-labels:
-  app: mta
```
Comment (gabriel-farache) [inline charts/workflows/charts/mta/values.yaml:6]: You bumped and cleaned in this PR, right? I mean the labels is not used so you removed it

Comment (masayag) [reply charts/workflows/charts/mta/values.yaml:6]: yes. labels aren't used in the template.

Comment (gabriel-farache) [reply charts/workflows/charts/mta/values.yaml:6]: would it possible to have 2 commits: 1 for clean and 1 for version bump?

Sorry for asking

Comment (masayag) [reply charts/workflows/charts/mta/values.yaml:6]: it is possible, but brings more question:
- should there be a commit to update each chart? 
- should there be a different commit to update the parent chart?

Let's have discussion for it and add conclusion to the CONTRIBUTION.md (after this PR is merged ;-) )

Comment (gabriel-farache) [reply charts/workflows/charts/mta/values.yaml:6]: ack

My main concern is separation of concern, so when you look at the history you know what was changed

This is actually related to the "good practices" we would want to enforce so yeah the contributing is the perfect place for that

**Review by gabriel-farache — APPROVED** [review]

**Pull request merged**